### PR TITLE
Improve API documentation navigation and function listing

### PR DESCRIPTION
**Pull request by cloudyday**
- List available "find functions" in `__all__` assetcentral package `__init__.py`
- Now these imports will show up in the package documentation
- Show packages instead of modules in the API documentation, so that the reader can easily find the "find functions" in the assetcentral package documentation
- Fix/add LICENSE information

### CODE REVIEW

**Comment by cloudyday** [comment]
Hello again @SebastianWolf-SAP :-) 
I'm not 100% sure if the licensing with REUSE is done right here. 

Could you have a look at that last commit and let me know?

File: .reuse/dep5 [hunk 1]
```diff
@@ -27,3 +27,7 @@ Disclaimer: The code in this project may include calls to APIs (“API Calls”)
 Files: *
 Copyright:  2020-2021 SAP SE or an SAP affiliate company and project "Sailor" contributors
 License: Apache-2.0
+
+Files: docs/_templates/apidoc/module.rst_t, docs/_templates/apidoc/package.rst_t
+Copyright:  2007-2021 by the Sphinx team.
```
Comment (SebastianWolf-SAP) [inline .reuse/dep5:32]: If I recall correctly, you mentioned that you modified their original code. In this case, that should also be mentioned here:

```suggestion
Copyright:  2007-2021 by the Sphinx team. Modifications 2021 SAP SE or an SAP affiliate company and project "Sailor" contributors
```

**Review by SebastianWolf-SAP — CHANGES_REQUESTED** [review]
I'd do a minor change to the copyright notice (see my suggestion). Moreover, please add the licensing section to your README. I guess, you're good to go afterwards.

**Comment by cloudyday** [comment]
Thanks for the suggestions @SebastianWolf-SAP . I also saw that our copyright information was missing from the LICENSE file in the project root. I added that. 

Could you please check again?

**Review by SebastianWolf-SAP — APPROVED** [review]
LGTM, thanks for the changes.

**Review by atrejut — APPROVED** [review]

**Pull request merged**